### DISCUSSION TITLE
Add missing axis unit Megawatt-hour (MWh)

### DISCUSSION THREAD

**Discussion by marians** [grafana/grafana#65771] [Dashboards, Panels & Visualizations]
I would like to visualize power generation data with a source unit of Megawatt-hour (MWh) in a time series graph in a dashboard.

The **Standard options** > **Unit** menu offers me, in the **Energy** category:

- Watt-hour (Wh)
- Kilowatt-hour (kWh)

Currently my use case requires adding a multiplication by factor 1000 on the query level to convert from MWh to kWh. Then the unit display changes to MWh and GWh as expected. It's simple once you know how to do it (had to find out how), so adding the unit could make the system a bit easier to use for noobs like me.

I'm using Grafana v9.4.7.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82770. Feel free to continue the discussion around this there. Thank you!
